src/index.js

```diff
@@ -58,6 +63,71 @@ function errorFlowCouldNotRun(loc) {
   };
 }
 
+function createFilteredErrorRule(filter: (CollectOutputElement) => any) {
```

Comment:
Is there a more specific type we can use than `any`?

Reply:
Ideally, we'd use boolean.  But I know that people like to use true-ish things in place of actual booleans for filter functions.